release-notes-4.0.0.md

```diff
@@ -1,9 +1,8 @@
-!not-ready-for-release!
-
 #### Version Number
 ${version-number}
 
 #### New Features
+- None
```

Comment:
Should this bug fix be in the release notes? https://github.com/CAFAudit/audit-service/commit/062eed15ed0a559b57adc2ecd884b99fde9b0864

Reply:
I don't think so, it only affected builds, not actual clients of this repo